index.md

```diff
@@ -1096,42 +1102,43 @@ Basic Commands (Beginner):
 ..
 ```
 
-Lots of interesting things under the ellipses but honestly, do we really feel like reading beyond **run**?
+Lots of interesting things under the ellipses but honestly, do we really feel like reading beyond **create**?
 
 ```console
-$ kubectl run
-Error: required flag(s) "image" not set
+$ kubectl create --help
+Create a resource from a file or from stdin.
+
+ JSON and YAML formats are accepted.
 
 Examples:
-  # Start a single instance of nginx.
-  kubectl run nginx --image=nginx
-..
+  # Create a pod using the data in pod.json.
+  kubectl create -f ./pod.json
+
+  # Create a pod based on the JSON passed into stdin.
+  cat pod.json | kubectl create -f -..
```

Comment:
Suggestion here:
Maybe include:
```
Available Commands:
  clusterrole         Create a ClusterRole.
  clusterrolebinding  Create a ClusterRoleBinding for a particular ClusterRole
  configmap           Create a configmap from a local file, directory or literal value
  cronjob             Create a cronjob with the specified name.
  deployment          Create a deployment with the specified name.
...
```
Something like: "deployment" is exactly what we are doing!
Then you can include:
```
ubectl create deployment --help
Create a deployment with the specified name.

Aliases:
deployment, deploy

Examples:
  # Create a new deployment named my-dep that runs the busybox image.
  kubectl create deployment my-dep --image=busybox
```
and continue with  line 1121: "Absolutely fucking beautiful! 😍"